arches_querysets/querysets.py

```diff
@@ -206,6 +168,55 @@ def _clone(self):
         clone._as_representation = self._as_representation
         return clone
 
+    def _get_node_value_for_python_annotation(self, tile, node, tile_val):
+        datatype_instance = DataTypeFactory().get_instance(node.datatype)
+
+        if self._as_representation:
```

Comment:
We're in the queryset layer (core Django) here. But if the queryset has been told (e.g. by the REST serializers) that we're going straight to JSON, then we go straight there, and skip making python instances for things.